### PR TITLE
chore: setup Tailwind CSS and daisyUI

### PR DESCRIPTION
**Pull request by kobakaku**

### CODE REVIEW

File: projects/web/src/styles.scss [hunk 1]
```diff
@@ -1 +1,3 @@
-/* You can add global styles to this file, and also import other style files */
+@import 'tailwindcss/base';
+@import 'tailwindcss/components';
+@import 'tailwindcss/utilities';
```
Comment (YasunoriMATSUOKA) [inline projects/web/src/styles.scss:3]: image

**Review by YasunoriMATSUOKA — CHANGES_REQUESTED** [review]
You need to change styles.scss.

**Comment by kobakaku** [comment]
You mean I should change scss to css ?

File: projects/tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ['./web/src/**/*.{html,ts}', './admin/src/**/*.{html,ts}', './lp/src/**/*.{html,ts}'],
```
Comment (YasunoriMATSUOKA) [inline projects/tailwind.config.js:3]: I guess tailwind.config.js should be repository root.
Remove this and `npx tailwindcss init` on repository root directory.
And, these paths should be `'./projects/web/src/**/*.{html,ts}', './projects/admin/src/**/*.{html,ts}', './projects/lp/src/**/*.{html,ts}'`

**Review by YasunoriMATSUOKA — CHANGES_REQUESTED** [review]
tailwind.config.js should be changed

**Comment by YasunoriMATSUOKA** [comment]
> You mean I should change scss to css ?

No. In the scss file, css works. I think you don't need to convert it.

**Comment by kobakaku** [comment]
I just changed tailwind.config.js.
but, as you may know, @tailwind directrive is'nt recognized on scss file

**Comment by YasunoriMATSUOKA** [comment]
Did it work or not?
And, I guess you also need to change style.scss in project admin and lp.

**Comment by YasunoriMATSUOKA** [comment]
> I just changed tailwind.config.js.
> but, as you may know, https://github.com/tailwind directrive is'nt recognized on scss file

In my another project it worked.
Did you check on your environment to work or not?
image

**Comment by kobakaku** [comment]
> In my another project it worked. Did you check on your environment to work or not? image
Yeah I tryed it, but it didn't work..

**Comment by kobakaku** [comment]
スクリーンショット 2023-02-16 15 58 23

**Comment by YasunoriMATSUOKA** [comment]
Oh, you mean lint warning? But it works as the following and we don't edit scss anymore...

`app.component.ts`
image

`screenshot`
image

**Comment by YasunoriMATSUOKA** [comment]
OK, I guess I can understand you wanna say. 

css way ... tailwind style works but lint warning occurs.

```
@tailwind base;
@tailwind components;
@tailwind utilities;
```

scss way ... tailwind style works and lint warning doesn't occur.

```
@import 'tailwindcss/base';
@import 'tailwindcss/components';
@import 'tailwindcss/utilities';
```

You can change style.scss to scss way.
Is it fine?

And you also need to fix admin and lp

**Comment by kobakaku** [comment]
> Oh, you mean lint warning?

yeah.

Oh really... your environment does it correctly.
Im using  tailwindcss (v3.2.6).

**Comment by kobakaku** [comment]
I dont know the reason but it also did work on my environment.
And, I will continue to do scss way!!!

**Comment by kobakaku** [comment]
I completed the issue.
Please review my commit.

**Review by YasunoriMATSUOKA — APPROVED** [review]
LGTM

**Pull request merged**